Development-Guidelines/building-angular-project.md

```diff
@@ -1,95 +1,95 @@
-
-So see it as an additional powerful tool in your arsenal, once you see the power, you don't want to go back.
+# Working with the backoffice UI AngularJs project 
+
+## Overview
+Umbraco 7 has a slightly unorthodox project structure, compared to a normal ASP.NET project. This is by design, a choice from the beginning to embrace a much larger group than "just" the developers who know how to use Visual Studio. 
+
+As a result, the Umbraco UI is not a Visual Studio project, but simply a collection of folders and files, following certain conventions, and a small configuration file called `gulpfile` - we will get to the gulp part in a moment. 
+
+This means that anyone with a text editor can open the UI source, make changes and run the project, without having Visual Studio installed - we will get into how to do that in a moment as well. 
+
+The bottom line is the UI project has zero dependencies on ASP.NET or Windows. However, you will need Node.js installed, but don't worry we will get into that in a second.
+
+
+## Prerequisites
+Umbraco 7 needs a couple of things to run:
+
+### Node.js 
+To compile and run the UI project you need [Node.js](https://nodejs.org) installed, it is needed for both Windows and OSX.
+
+### gulp
+When you have Node.js installed, you need to install gulp. gulp is a simple JavaScript task runner, basically like NAnt, MSBuild or any traditional build system [more about gulp here](https://gulpjs.com).
+
+To install, open a terminal and run: 
+	
+	npm install gulp-cli -g
+
+For OSX users, you will most likely need to do: 
+
+	sudo npm install gulp-cli -g
+
+This installs a `gulp` command into your terminal so you can run gulp scripts with simple commands. That might sound scary, but really it isn't, while working with Umbraco 7, you will become really good friends with `gulp` and your terminal. 
+
+### Project dependencies
+Now its time to install all the dependencies that the project requires to compile, debug, test, minify and so on. Luckily this is all automatic and is done with the Node.js package manager (which you already have installed with node)
+
+In your terminal, browse to the `Umbraco.Web.Ui.Client` folder and run the command: 
+
+	npm install
+
+This will output a ton of feedback in your terminal when it stops, your project is ready to run. 
+
+This might seem like a lot of stuff to do, but think of it this way, every time you set up this environment, all you have to do is run `npm install` and everything will be running smoothly.
+
+## Running from source without Visual Studio or IIS
+The Umbraco 7 project includes a complete mocked data model and an embedded web server to run off. 
+
+So to get the project up and running in a browser as fast as possible, open a terminal, browse to `Umbraco.Web.Ui.Client` folder and run the command: 
+
+	gulp dev
+
+this will do the following: 
+
+- Compile and merge the project files
+- Compile fewer files to one .css file

```

Comment:
```suggestion
- Compile LESS files to one .css file
```
I see the confusion here, we are actually using the css library LESS http://lesscss.org/ 😄